math/mean_test.go

```diff
@@ -0,0 +1,51 @@
+package math
```

Comment:
```suggestion
package math_test
```

Reply:
Does this indicate of creating another directory for math_test ?

Reply:
> Does this indicate of creating another directory for math_test ?

Or just moving the whole test script to the math_test.go under math package?

Reply:
Done updating the package.